docs/index.md

```diff
@@ -11,15 +11,40 @@ This is the documentation for the `cfgov-refresh` project, a redesign of the [ww
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
+- [Apache]() - Version? Which project exactly? https://apache.org/index.html#projects-list
 - [Homebrew](https://brew.sh) - package manager for installing system software on OSX
-- Python and PIP (Python package manager)
-- [Jinja templates](http://jinja.pocoo.org) for front-end rendering
-- [Wagtail CMS](https://wagtail.io) for content administration
-- Dependencies, listed below
+- Python 2.7 and PIP (Python package manager)
+- [Jinja2 templates](http://jinja.pocoo.org/docs/2.10/) for front-end rendering. See [`requirements/libraries.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/libraries.txt) for version.
+- [Wagtail CMS](https://wagtail.io) for content administration. See [`requirements/wagtail.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/wagtail.txt) for version.
+- [PostgreSQL](https://www.postgresql.org/) and [Psycopg](http://initd.org/psycopg/) for our database. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for version.
+- Additional dependencies, listed below
 
-# Dependencies
+# Additional dependencies
 
 - [Elasticsearch](https://www.elastic.co):
   Used for full-text search capabilities and content indexing.
-- [Node](http://nodejs.org) and [yarn](https://yarnpkg.com/):
-  Used for downloading and managing front-end dependencies and assets.
+- [Node 8](http://nodejs.org) and [yarn](https://yarnpkg.com/):
+  Used for downloading and managing front-end dependencies and assets. Front-end dependencies are listed in the project's [package.json](https://github.com/cfpb/cfgov-refresh/blob/master/package.json) file.
+- [Gulp 4](https://gulpjs.com/)
+- [virtualenv](https://virtualenv.pypa.io/en/stable/)
+- [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
```

Comment:
These last two are not technically necessary if you're using the Docker setup, but this is fine until we commit to one direction or the other.